src/main/java/duke/logic/commands/RemindersCommand.java

```diff
@@ -2,12 +2,59 @@
 
 import duke.exceptions.DukeException;
 import duke.models.LockerList;
+import duke.models.locker.Address;
+import duke.models.locker.Locker;
+import duke.models.locker.SerialNumber;
+import duke.models.locker.Zone;
+import duke.models.tag.Tag;
 import duke.storage.Storage;
 import duke.ui.Ui;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+import java.util.stream.Collectors;
+
 public class RemindersCommand extends Command {
+
+    public static final String COMMAND_WORD = "reminders";
+
+    public RemindersCommand() {
+
+    }
+
     @Override
     public void execute(LockerList lockerList, Ui ui, Storage storage) throws DukeException {
 
+
+        LocalDate now = LocalDate.now();
+        String dateNow = now.format(DateTimeFormatter.ofPattern("dd-MM-uuuu"));
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd-MM-uuuu");
+        LocalDate localDateNow = LocalDate.parse(dateNow,formatter);
+
+        FindCommand.FindLocker unauthorisedLocker = new FindCommand.FindLocker();
```

Comment:
can explain me the use of using findLocker here?

Reply:
to search for unauthorised lockers/broken lockers you can just to the following
lockerList.getLockerList().stream(p -> p.getTag().equals(new Tag("unauthorized")
similarly  for broken.
Dont actually need findLocker